Add bundle method

SimenB opened the pull request:
## JIRA Issue
[COREWEB-10](https://jira.finn.no/browse/COREWEB-10)

## Status
**READY**

## Description
<!-- A few sentences describing the overall goals of the pull request's commits. -->
This adds a new method called `bundle` on the exported class. This make the module match the API of the JS writer.

In particular it enables us to instantiate a `CssWriter` a single time, but consume from it multiple times.

## Todos
- [x] Tests
- [x] Documentation

## Deploy Notes
<!-- Notes regarding deployment of the contained body of work. These should note any
db migrations, etc. -->

## Related PRs
* None

SimenB committed:
feat: Add `bundle` method

Adds a new `bundle` method to match the API of the JS writer. This allows you to instantiate a
writer a single time, but consume from it multiple times

BREAKING CHANGE: The user must now call `.bundle` before the writer can be consumed

digitalsadhu commented on README.md:
```diff
@@ -82,12 +82,12 @@ const writer = new CssWriter([
-handler:
+The writer is an event emitter, which has a method called `bundle`, which
+returns a readable stream in object mode so in order to access the data you may
+register a data handler and listen for objects to be passed to the handler:
```
Is this right? Shouldn't bundle return a stream that outputs string chunks of bundled css?
I thought that the js writer was a feed stream but if you call .bundle() then you get a stream that streams the bundled js text?

SimenB replied:
We discussed this IRL

digitalsadhu reviewed (commented):
Question there about correct usage of bundle()

SimenB committed:
fix: change bundle's id to just be hash of content

Previusly the hash was determined by the name and version of the module, in addition to the path to
and content of the file. We now just look at the content

BREAKING CHANGE: Hashing stretegy has changed

digitalsadhu reviewed (approved):
LGTM
